### DISCUSSION TITLE
Quake crashing

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#354] [Q&A]
![IMG_20230815_152010~2](https://github.com/eduardofilo/RG350_adam_image/assets/96494219/9329390b-669b-4bf8-af4a-412cc44d7b10)

I have seen 3 other people reporting on Reddit, Quake crashing in exactly the same place as I am experiencing a crash, really early in the first level...after raising up a platform to traverse some acid (probably)...cross over, turn corner...& black screen, exit out to simple menu, every time. On a 350m using latest Adam image, 2.1.  Seemed like Mission pack 1 was OK...but now that is consistently crashing in the same spot, a few levels in...when opening a hatch with a silver key, under water...having turned a force bridge on a bit earlier. Any one else experiencing these instances?

**Comment by Ganimoth** [comment]
Yeah, that is a common problem, both Tyrquake and RGQuake crash in the same spots

**Reply by chrisstonehill** [reply]
Any solutions? Mine might just be playing the Remaster (only just got it...& the brand new Quake 2 one...oh my goodness!) on my laptop & monitor. A LOT easier than trying to aim with the 350m sticks. WOULD be nice to have Quake in my pocket though. Quake 2 doesn't have issues like this, playing it on the 350m?

**Reply by Ganimoth** [reply]
Sadly I havnt found any :(. I dont know about Q2, I am on RG280V, so Quake 2 is impossible to control

**Reply by chrisstonehill** [reply]
That's a shame. Yes, Q2 on 280v would be quite challenging I imagine 🤣...only works with twin sticks? Q1 on my 350m controls with twin sticks...but also possible with dpad & some of the face buttons for looking up & down?...or something? That doesn't sound so easy either. Seemed like Quake - Mission Pack 1 was going to be fine. Played for maybe about an hour...then crashing in the same spot over & over again...sigh 😔

**Reply by Ganimoth** [reply]
It would have to be specifically redone for 280v as Quake 2 OD port also wants SEPARATE bindings for menu buttons, so if you somehow manage to cram game controls to somewhat play, you dont have buttons for menu controls, so no saving for you! :D

**Reply by chrisstonehill** [reply]
Quake 2 was the last game I began playing, on my family PC...when it was released...before about 20 years of almost no gaming! Playing the remaster of it now feels amazing. I actually barely played it back in the 90s & my 7 years younger cousin got it from me. Sounds like he had a great time playing it on his family's PC.
